ui/app/templates/rules.html

```diff
@@ -205,6 +205,11 @@ <h5 title="Fallback Mapping" ng-show="rule.fallbackMapping || (show_sc && rule.s
             </a>
           </span>
       </h5>
+      <span ng-show="(rule.fallbackMapping || (!rule_id && show_sc && rule.scheduled_change !== null && rule.scheduled_change.fallbackMapping !== null )) && (rule.backgroundRate !== 100 || rule.scheduled_change.backgroundRate !== 100)" >
```

Comment:
Question from who has no experience with frontend:

Can expressions like bellow, be abstracted as controller functions?
`(rule.fallbackMapping || (!rule_id && show_sc && rule.scheduled_change !== null && rule.scheduled_change.fallbackMapping !== null )) && (rule.backgroundRate !== 100 || rule.scheduled_change.backgroundRate !== 100)`

On this page there is 5 tests like `!rule_id && show_sc && rule.scheduled_change !== null`

Reply:
Yeah, it could be moved to a function defined in $scope, and the function could be called from there. That's not a blocker IMO. Sometimes we avoid doing that because there are not many instances of exactly the same condition (eg: it could look at different fields, or maybe one of the parts is removed). I don't feel strongly about this.